src/main/java/com/poolapp/pool/model/Booking.java

```diff
@@ -0,0 +1,49 @@
+package com.poolapp.pool.model;
+
+import com.poolapp.pool.model.enums.BookingStatus;
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "booking", uniqueConstraints = @UniqueConstraint(columnNames = {"user_id", "session_id"}))
```

Comment:
uniqueConstraints not convenient here, the better to use this for certain fields

Reply:
like here https://github.com/esfagno/booking1/pull/3/files#diff-4f7d01febedd5e8c605376f8bc52f774de6bf0bf781fa0b1e009b59e1b061b22R40